code/js/modules/Sitelist.js

```diff
@@ -114,6 +114,7 @@
       "music.yandex": {name: "Yandex", url: "http://music.yandex.ru", controller: "YandexController.js"},
       "radio.yandex": {name: "Yandex Radio", url: "http://radio.yandex.ru", controller: "YandexRadioController.js"},
       "youarelistening": {name: "YouAreListening.to", url: "http://www.youarelistening.to", controller: "YouarelisteningtoController.js"},
+      "xiami": {name: "Xiami", url: "http://www.xiami.com", controller: "XiamiController.js"},
```

Comment:
The `controller` property here is unnecessary since the file naming adheres to pattern, please remove! Everything else looks good to me